app/models/wiki_page.rb

```diff
@@ -94,6 +95,10 @@ def search(params)
         q = q.where("is_deleted = false")
       end
 
+      if params[:parent].present?
+        q = q.where("parent LIKE ? ESCAPE E'\\\\'", params[:parent].downcase.strip.tr(" ", "_").to_escaped_for_sql_like)
+      end
```

Comment:
```suggestion
      q = q.attribute_matches(:parent, params[:parent].try(:tr, " ", "_"))
```

https://github.com/e621ng/e621ng/blob/8edb1b9226d63aeb405aedd596af2956317d3ae1/app/models/application_record.rb#L105

The `attribute_matches` does the same thing internally for text columns, additionally it will use the tsvector when wildcards aren't present